Include 'all' as a field attribute of ManyToMany relationships.

YPCrumble opened the pull request:
Thanks for maintaining this great repo! This is really an issue, but the fix is easy so thought it better to post as a PR. If I've missed something feel free to close. 

Per [the docs](https://docs.djangoproject.com/en/3.1/topics/db/examples/many_to_many/) It's possible to call `all()` on a many to many attribute. For example, the docs call `a1.publications.all()` in their example.

I'm seeing this error pop up which I don't believe is a configuration error.

atodorov reviewed (changes requested):
Change looks good but you need to add the tests for it as well.

YPCrumble commented:
@atodorov thank you! Added the test - I believe this is correct but let me know if it's not? Also added to the CHANGELOG for release today but happy to remove or edit that if you'd prefer.

YPCrumble commented:
@atodorov I'm actually going to close this for now - I'm not 100% sure it's necessary as this is actually an issue with my factories, not a plain Django model.
